CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
-# v0.1.0
+# 0.2.0
+
+## Fixes
+
+* Correctly configure OAuth2 authentication
```

Comment:
"Correctly configure in-cluster token-based authentication"

Reply:
Ah good catch